src/languages/luau.ts

```diff
@@ -0,0 +1,15 @@
+// This is a child_process
+import luau from "../luau";
+
+luau.print = console.log;
```

Comment:
Why? Make your own function that logs output instead of having to do something hacky with stdout/err.

Reply:
luau.print is a module level function, meaning I am unable to differenciate what output is for who. This is the solution I thought of, but if you have a better idea I'd love to hear it.